### PR TITLE
Fix/timeseries visualization api on review

### PR DESCRIPTION
**Pull request by takam5f2**
## Description

I merged updates on https://github.com/tier4/CARET_doc/pull/119 to this branch.

## Pre-review checklist for the PR author

- [x] I've confirmed the [contribution guidelines](https://github.com/tier4/caret/blob/main/.github/CONTRIBUTING.md).
- [x] The PR is ready for review.

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR has been reviewed.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

### CODE REVIEW

**Comment by takam5f2** [comment]
@atsushi421  Could you check this PR?

File: docs/gallery.md [hunk 1]
```diff
@@ -5,61 +5,61 @@ If you find what you are interested in, please access to a link embedded in a he
...
 
 ![callback_period_df](./imgs/callback_period_df.png)
 
-#### Time-line graph
+#### Time-series graph
 
 ![callback_period_time_line](./imgs/callback_period_time_line.png)
 
-### [Execution time (latency)](./visualization/latency/latency.md#callback)
+### [Execution time (latency)](./visualization/latency/index.md#callback)
```
Comment (atsushi421) [inline docs/gallery.md]: Latency (or Execution time) lools better.

File: docs/visualization/latency/index.md [hunk 2]
```diff
@@ -46,7 +45,6 @@ The vertical axis means execution time of callback function, labeled as `Latency
 ## Communication
 
 `Plot.create_latency_timeseries_plot(communications: Collection[Communication])` is called when you are concerned how long time is consumed from message publish to corresponding subscription.
```
Comment (atsushi421) [inline docs/visualization/latency/index.md:47]: It is better to insert the following sentences as well as other parts.

> Here, CARET takes into account communication when both transmission and reception on a message are performed successfully without being lost.
> See [Premise of communication](../premise_of_communication.md) for more details.

**Review by atsushi421 — CHANGES_REQUESTED** [review]
@takam5f2 
Sorry for the late review.
Please make some quick fixes.

File: docs/visualization/latency/index.md (hunk 2 above)
Comment (takam5f2) [reply docs/visualization/latency/index.md:47]: Fixed at https://github.com/atsushi421/CARET_doc/pull/1/commits/ff485321764509954a3b9910c772718427a01976

File: docs/gallery.md (hunk 1 above)
Comment (takam5f2) [reply docs/gallery.md]: Fixed at https://github.com/atsushi421/CARET_doc/pull/1/commits/ff485321764509954a3b9910c772718427a01976

**Comment by takam5f2** [comment]
@atsushi421  
Could you check the PR again?

**Review by atsushi421 — APPROVED** [review]
LGTM!

**Comment by takam5f2** [comment]
@atsushi421  
Could you merge this PR to your branch? I don't have permission to write your repository. Thanks,

**Pull request merged**